tools/verifier-helper/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "verifier-helper"
```

Comment:
Actually, I would place it in `packages/helpers`, call the create `lavs-helpers` and put the code in `src/verifier.rs`, with a `pub mod verifier`, so it gets imported as: `use lavs_helpers::verifier::ensure_valid_vote` or whatever

Reply:
cool, this opens up `helpers` to hold way more than the verifier functionality. 

Fixed in 813bdb3dc943